docs/yul.rst

```diff
@@ -1162,7 +1162,7 @@ An example Yul Object is shown below:
         code {
             function allocate(size) -> ptr {
                 ptr := mload(0x40)
-                if iszero(ptr) { ptr := 0x60 }
+                if iszero(ptr) { ptr := 0x80 }
```

Comment:
I think we may want also to update a code snippet in line 1194.

Reply:
I'd suggest to revert the change, i.e. change it back to ``0x60``, but instead add a comment like:
```
// Note that Solidity generated IR code reserves memory offset ``0x60`` as well, but a pure Yul object is free to use memory as it chooses.
```
or something like that. That may clear things up, and then *explicitly* clearify what's happening?

Reply:
@0xpranay would you still like to apply the changes?

Reply:
Yes. Will do soon.

Reply:
@leonardoalt applied the changes.

Reply:
Great, thank you!